Task_TheGridProblem/index.html

```diff
@@ -1,11 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
```

Comment:
Check if the path to the CSS file (index.css) is correct. Make sure the file exists in the specified location.